lib/run_printer.py

```diff
@@ -161,13 +165,17 @@ async def dump_run(args):
 
     sleeper = BackoffSleeper(jitter=random.random())
     check_run = get_run_checker()
+    out_file = args.out_file.resolve() if args.out_file else None
+    if out_file and out_file.is_dir():
+        logging.error("--out-file flag expects a file path and not directory")
+        sys.exit(0)
```

Comment:
Should be `sys.exit(1)`

Reply:
Removed this check from this file as suggested by Kareem.